egs2/chime7_task1/asr1/run.sh

```diff
@@ -120,16 +128,20 @@ if [ ${stage} -le 1 ] && [ $stop_stage -ge 1 ]; then
           continue # dipco has no train set
       fi
 
-      if [ -n "$decode_only" ] && [ $dset_part != "$decode_only" ]; then
+      if [ "$decode_train" != "train" ] && [ $dset_part != "$decode_train" ]; then
         continue
       fi
 
-      if [ ! -d $chime7_root/$dset/audio/$dset_part ]; then
-        log "Skipping $dset $dset_part because it does not exist on disk. This is
-        fine if you don't have evaluation set yet."
+      if [ "$decode_train" == "train" ] && [ $dset_part == "eval" ]; then
         continue
       fi
 
+      #if [ ! -d $chime7_root/$dset/audio/$dset_part ]; then
+      #  log "Skipping $dset $dset_part because it does not exist on disk. This is
+      #  fine if you don't have evaluation set yet."
+      #  continue
+      #fi
+
```

Comment:
Do you want to keep these lines?

Reply:
We can remove it. 
I have added a new flag for skipping the evaluation. 
Let me update the PR